api/types/types.proto

```diff
@@ -2991,6 +2991,12 @@ message SessionTrackerSpecV1 {
     // creation.
     repeated SessionTrackerPolicySet HostPolicies = 16
         [ (gogoproto.jsontag) = "host_roles,omitempty" ];
+
+    // DatabaseName is the database server this session belongs to.
+    string DatabaseName = 17 [ (gogoproto.jsontag) = "database_name,omitempty" ];
+
+    // AppName is the app server this session belongs to.
+    string AppName = 18 [ (gogoproto.jsontag) = "app_name,omitempty" ];
```

Comment:
What about `DesktopName`?

Reply:
I'm using `Address: desktop.GetAddr()` for Desktop sessions, ssh sessions also use `Address`. It looks like the only other value which might be relevant is `desktop.GetDomain()`, or is there another value which would correspond to `DesktopName`?

Reply:
`desktop.GetMetadata().Name`?